### ISSUE TITLE
Support for ES2015/JSX

### ISSUE THREAD

**Issue by tf** [lipp/doclets#54]
The build for the following Webpack compiled project failed:

  https://doclets.io/tf/pageflow-react/add-to-doclets

It contains both ES2015 code and JSX files. Is support for those features planned? [JSDoc claims](https://github.com/jsdoc3/jsdoc/issues/1001) to support it.

**Comment by lipp** [comment]
Not sure about this. I'll have to investigate on it ...

**Comment by lipp** [comment]
This will definitely come. ES2015 and JSX are way too important to ignore !
Working on orga support now. This is next! @tf

**Comment by tf** [comment]
I also tried running JSDoc locally and I can confirm that it is not a doclets specific problem. ES2015/JSX support in JSDoc appears to be a work in progress still. So feel free to close or keep it open as a reference point for others...

**Comment by lipp** [comment]
thanks for digging into it! maybe the parser can handle it but the default rendering does not work for es2015? i am extracting the "pure" data from jsdoc and doing the rendering etc myself. hope this jsdoc es2015 thing will clear up in the next time.

**Comment by lipp** [comment]
@tf thinking about switching to documentation.js . Any experience with it?

**Comment by tf** [comment]
Played a bit with it. But not enough to offer any real insights here.
Sorry.

Gerhard Preuss <notifications@github.com> schrieb am Do., 24. März 2016,
12:54:

**Comment by lipp** [comment]
@tf it seems some es2015 stuff is already working: https://doclets.io/crawlkit/crawlkit/v2.0.1#dl-CrawlKit

using classes, getters / setters etc.

**Comment by felixfbecker** [comment]
[esdoc](https://esdoc.org/) aims to be a better JSDoc with proper support for ES2015

**Comment by lipp** [comment]
yes, esdoc might be a great choice for now.

**Comment by tf** [comment]
I gave it another try and with the `jsdoc` version currently linked in the `package.json` (which references the same commit as `master` at the moment), ES2015 and also JSX work ([example](https://doclets.io/tf/pageflow-react/add-to-doclets)). :+1: So maybe this issue can be closed?

As for `esdoc`, I took a closer look and I'd be careful to switch. To me it looks like the project is basically maintained by one person who seems to be very busy at the moment. There even is a custom "owner: busy" batch right at the top the README :joy:
